examples/recording_service/pluggable_storage/cpp/FileStorageReader.cxx

```diff
@@ -220,11 +224,6 @@ bool FileStorageStreamReader::read_sample()
             throw std::runtime_error(
                     "Failed to read current data.msg field from file");
         }
-        std::getline(*data_file_, prefix);
```

Comment:
What did this fragment even attempt to do?

Reply:
The problem was that we have two formats. The C format was storing the samples without an empty line between them and the C++ was storing the samples using an empty line between them. I faced some issues related to keeping this extra empty line on X64Win64VS2015. Due to that and for consistency purposes I prefer to use the C format.